api/features/feature_segments/tests/test_views.py

```diff
@@ -134,49 +132,32 @@ def test_delete_feature_segment(segment, feature, environment, client):
     assert not FeatureSegment.objects.filter(id=feature_segment.id).exists()
 
 
-@pytest.mark.parametrize(
-    "client", [lazy_fixture("master_api_key_client"), lazy_fixture("admin_client")]
-)
-def test_audit_log_created_when_feature_segment_created(
```

Comment:
I removed these as I don't really see that we need an audit log created based on FeatureSegment creation. It's only important (I think) to audit when a segment override feature state is created.